### PR TITLE
Temporarily skip IBCMerge optimizations for Microsoft.CodeAnalysis.Wo…

### PR DESCRIPTION
**Pull request by mavasani**
…rkspaces

ngen is crashing on the current IBCMerge data for this assembly and corrupting the generated native image, which is subsequently causing an intermittent CLR crash (VSO #290723).
We are temporarily disabling IBCMerge for this assembly, until the CLR bug is fixed.

### CODE REVIEW

**Comment by mavasani** [comment]
Tagging @dotnet/roslyn-perf @dotnet/roslyn-infrastructure for review

@MattGertz for ask mode approval.

File: src/Workspaces/Core/Portable/Workspaces.csproj [hunk 1]
```diff
@@ -13,6 +13,8 @@
     <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <ServiceablePackage>true</ServiceablePackage>
+    <!-- Temporary workaround for CLR bug, VSO #290723. Please remove the below line once this bug is fixed. -->
```
Comment (masterlor) [inline src/Workspaces/Core/Portable/Workspaces.csproj]: https://github.com/dotnet/roslyn/pull/15193.patch

Comment (masterlor) [inline src/Workspaces/Core/Portable/Workspaces.csproj]: My first build, thanks for the help

**Comment by mavasani** [comment]
Ping

**Comment by mavasani** [comment]
Merging it in to unblock DDRITs.

**Comment by ManishJayaswal** [comment]
👍

**Pull request merged**